### PR TITLE
chore: create CODEOWNERS

### PR DESCRIPTION
**Pull request by vapurrmaid**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+@coder/coder-docs
```
Comment (jawnsy) [inline .github/CODEOWNERS:1]: does this ping coder-docs for all changes in this repo?

**Review by jawnsy — APPROVED** [review]
LGTM, just had a minor question

File: .github/CODEOWNERS (hunk 1 above)
Comment (vapurrmaid) [reply .github/CODEOWNERS:1]: Yea, which might not be the best granularity (we may want to separate owners for CI or other aspects). Very open to ideas, I opened this to start that convo ✨

**Review by khorne3 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (khorne3) [reply .github/CODEOWNERS:1]: I think it should be fine? It's just us at this moment and I can handle the (infrequent CI) stuff. It's not the busiest of repos

**Pull request merged**